test/util/testnode/full_node.go

```diff
@@ -226,6 +228,27 @@ func DefaultNetwork(t *testing.T, blockTime time.Duration) (accounts []string, c
 		require.NoError(t, cleanupGRPC())
 	})
 
+	return cctx, tmCfg.RPC.ListenAddress, appCfg.GRPC.Address
+}
+
+// DefaultNetwork starts a single valiator celestia-app network using test
+// friendly defaults. These defaults include all default values and genesis
+// params, modified for fast block times and 300 funded accounts. The returned
+// client.Context has a keyring with all of the funded keys stored in it, which
+// can be accessed using the returned accounts.
+func DefaultNetwork(t *testing.T) (accounts []string, cctx Context) {
+	// we create an arbitrary number of funded accounts
+	accounts = make([]string, 300)
+	for i := 0; i < 300; i++ {
+		accounts[i] = tmrand.Str(9)
+	}
+
+	tmCfg := DefaultTendermintConfig()
+	tmCfg.Consensus.TargetHeightDuration = time.Millisecond * 400
```

Comment:
We could probably get away with doing this quicker. It means that certain height dependent tests will operate faster

Reply:
iirc, I think we increased it to 400 because of CI flake issues, but I don't remember if this actually fixed anything so we can try them at 1ms as it would certainly be nice

c5c09b3a23e483648c5cbc03501817fbcc03a66e